cmd/osv-scanner/main.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 var (
-	version = "dev"
+	// Update this variable when doing a release
+	version = "1.2.0"
```

Comment:
Should this have a "-dev" suffix by default in non-release build contexts?

Reply:
Thought about that a bit, I feel like it probably should not, since it seems like all linux package repos will build their own version, the version of osv-scanner people get from them will then also have the -dev suffix, which I don't think is ideal.

The user can already tell if it's built by us directly or not by looking at whether "built on" and "commit" is filled out or not.

Reply:
SGTM, that makes sense.